tests/integration/test_integration_platform_doc.py

```
@@ -1145,73 +1211,263 @@ def test_complete_workflow(self, local_docs_dir, temp_dir, agent_config):
         db_path = str(Path(temp_dir) / "store")
 
-        # Step 1: Initialize platform docs
         cfg = DocumentConfig(
             type="local",
             source=str(local_docs_dir),
             version="v1.0",
             chunk_size=256,
         )
 
-        init_result = init_platform_docs(
+        init_result, nav_result, search_result, doc_result = _run_e2e_workflow(
```

Ruff (0.15.0)

[warning] Unpacked variable `init_result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)